src/main/java/frc/robot/commands/BallChucker9000Shoot.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import frc.robot.subsystems.BallChucker9000;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+
+public class BallChucker9000Shoot extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final BallChucker9000 BallChucker9000Subsystem;
+
+  /**
+   * Creates a new BallChucker9000Shoot.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public BallChucker9000Shoot(BallChucker9000 subsystem) {
+    BallChucker9000Subsystem = subsystem;
+    
+    addRequirements(BallChucker9000Subsystem);
+  }
+
+  @Override
+  public void initialize() {
+    
```

Comment:
Remember to fill this in. We probably want 2 commands: One with limelight input, the other that just takes a DoubleSupplier or `double speed`